test/tbb/test_join_node.h

```diff
@@ -26,6 +26,12 @@
 #endif
 #endif
 
+#ifdef __MIPS
+    #define MAX_TUPLE_TEST_SIZE_LOCAL 3
+#else
+    #define MAX_TUPLE_TEST_SIZE_LOCAL MAX_TUPLE_TEST_SIZE
+#endif
+
```

Comment:
We usually use `TBB_TEST_LOW_WORKLOAD` macro for similar things.

Reply:
Agree, look here please, this macro allows to reduce the amount of testing on MIPS and it already set: https://github.com/oneapi-src/oneTBB/blob/master/cmake/compilers/GNU.cmake#L54